### ISSUE TITLE
New copy: first draft

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/student-website#42]
TECH FOR GOOD PRODUCT DEVELOPMENT PROGRAMME

[ APPLY ] ( link to application form )

Get your MVP built for free

The Founders & Coders Tech for Good Product Development programme pairs nonprofit organisations and early-stage startups with a dedicated development team to build a mobile web application that delivers social impact through a new digital service. The programme is a one month commitment and there is no cost to participants.

What’s an MVP?
A minimum viable product. A product with just enough features to test out with real users, but small enough to build in a few weeks.

Do I have to be a non-profit?
Not necessarily, although it helps. Regardless of your legal status,  the problem you want to solve needs to address a social need.

What technologies do you use?
We build mobile web apps, primarily using JavaScript, Node.js and React.

Where will my development team be based?
In London and in Gaza.

What commitment do I have to make?
You will need to be available for approximately 8 hours per week for the 4 weeks of the programme. 

What technical support do you provide after delivery of the product?
Ongoing technical support is not provided after the end of the programme. 

Who owns the intellectual property to the work done?
The content remains the property of the product owner. The software will be released under the MIT open source licence.

What if I want to continue with development after the end of the month?
Members of the development team, both in London and Gaza, may be available to continue to work on a project on a freelance basis after the end of the Programme. 

How do you decide which product to work on?
We are primarily looking for a committed product owner and an interesting problem to solve with the potential to make a significant positive impact on its users.

How do I apply?
Complete the [application form]

**Comment by MissArray** [comment]
Suppose I own or represent a target organisation (start-up, for-profit, non-profit etc.). 
* I may, but will not necessarily be familiar with Agile terminology.
* You (FAC) need to mystify me sufficiently to convince me that you know your tech stuff, without, however, flummoxing me to the point that I feel perhaps a little stupid and perhaps a little irritated because you're making me feel a little stupid.

If you *first* address me in language I can understand and *then* throw in your tech terms, I think I'm 
* less likely to feel stupid 
* more likely to pay attention to what you're proposing
* more likely to be pleasantly mystfied by your esoteric terms

In other words, I wonder whether sort of 'reversing' the order in which our pitch is presented to organisations we're hoping to get interested to pitch in return, i.e. *from* plain speak *to* tech speak, rather than the other way around, might not make things easier all round?

**Comment by MissArray** [comment]
Suppose I own or represent a target organisation (start-up, for-profit, non-profit etc.).

    I may, but will not necessarily be familiar with Agile terminology.
    You (FAC) need to mystify me sufficiently to convince me that you know your tech stuff, without, however, flummoxing me to the point that I feel perhaps a little stupid and perhaps a little irritated because you're making me feel a little stupid.

If you first address me in language I can understand and then throw in your tech terms, I think I'm

    less likely to feel stupid
    more likely to pay attention to what you're proposing
    more likely to be pleasantly mystfied by your esoteric terms

In other words, I wonder whether sort of 'reversing' the order in which our pitch is presented to organisations we're hoping to get interested to pitch in return, i.e. from plain speak to tech speak, rather than the other way around, might not make things easier all round?

**Comment by thejoefriel** [comment]
Worth emphasising that it's free (like on the current site) as this will be a clear appeal. 

The Founders & Coders Tech for Good Product Development programme pairs nonprofit organisations and early-stage startups with a dedicated development team. Across this one-month programme, we will build for you a mobile web application that delivers social impact through a new digital service - all for FREE.

**Comment by MissArray** [comment]
Final copy - to be finalised. 

A sort of formatted version of the current draft [here:]
[https://docs.google.com/document/d/1yUGcobb06pzg76XcFrx3qbeiTH6GTg9j3EPH59PQYF4/edit?usp=sharing](url)


An MD version below:

**Final Copy - draft:**
 
 
 
**Founders and Coders Programme Name:**
***A pro-bono programme using technology to create positive social impact.***

**Your ideas can make the world a better place. We want to help.**
Our programme matches you up with a dedicated team of coders to build for your organisation a mobile web application that delivers positive social impact – for free.
 

**FAQs**
◼      What can I expect to get from participating in this programme?
We will work with you to clarify the issues you wish to solve, then build free web app – an MVP –  that meets these needs.
 
◼      *What is a mobile web app?*
A web app is a website that looks and feels like a mobile app. This means that your users don’t have to download anything from an app store. Web apps can be accessed by anyone using a web browser, on a computer, tablet or smartphone.
 
◼      *What’s an MVP?*
MVP means ‘minimum viable product’; i.e., a product with just enough features to test out with real users, but small enough to build in a few weeks.
 
◼      *What technologies do you use?*
We build mobile web apps, primarily using JavaScript, Node.js and React. Please ask if you’d like to know more.
 
◼      *Do I have to be a non-profit?*
We are open to working with any organisation that aims to make positive social change, regardless of its legal status. Usually, the clients we work with are non-profit organisations.
 
◼      *Where will my development team be based?*
Our developers are based in London, at Space4 [link], and Gaza, at Gaza Sky Geeks [link].
 
◼      *What commitment do I have to make?*
We’ll ask you to join us weekly at the Founders and Coders’ space in North London for the duration of the four-week programme. Spending approximately eight hours per week with us means we can develop your app dynamically together. There is no need to be based in London as long as you can meet this commitment.


 
◼      *What will I be doing during my visits?*
*On Site*
You will participate in an introductory workshop that will help us find out more about your needs.
You will be participating in weekly planning and reviewing sessions.
*Off site*
Morning calls with developers will provide an opportunity for feedback and guidance.
You’ll be reviewing development features and giving us feedback.
 
◼      *Who owns the intellectual property of the end product?*
You will retain all rights to any content used in the application (i.e. company logo, images, copy), while the application itself will be released under the MIT open-source licence [link]
 
◼      *What technical support do you provide after delivery of the product?*
Ongoing technical support is not provided after the end of the programme.
 
◼      *What if I want to continue with development after the end of the month?*
If you want to continue the project further, members of the development team, both in London and Gaza, may be available to continue to work on a project on a freelance basis after the end of the Programme.
 
◼     * How do you decide which products to work on?*
We are primarily looking for a committed product owner and an interesting problem to solve with the potential to make significant positive impact on its users.
 
◼      *How do I apply?*
Please complete the [application form].

**Comment by arrested-developer** [comment]
@thejoefriel @MissArray we did discuss as a group yesterday that the language 'free' might not always be the best to use. Yesterday our group came up with:

> Our programme matches you up with a team of our coders to build your organisation a mobile web application that delivers positive social impact.
> 
> Tech For Better is a one-month commitment that carries no financial cost.

Can we consider using language like 'pro-bono', 'no financial cost'? We do need to emphasise that there is a commitment to be made - also I might question the quality of a service that was offered 'for free'... The point is that we're willing to donate our time and resources to a project that creates a positive impact, right?

**Comment by sbinlondon** [comment]
I do feel that this draft has been changed from what we were working on a group yesterday. I thought we'd agreed on having a much warmer and more friendly tone - however, lots of these answers feel very brisk and so much "too the point" that we've lost the nice FAC tone we could have.

I think we also agreed that 'free' felt a bit gimmicky and we wanted to stick with phrases like 'pro-bono' as this captures the absent financial cost of what do yet gives it the sort of 'social impact' connotations we want.

**Comment by sima-qian** [comment]
Editing off @arrested-developer's suggestion,
> Our programme matches you up with a team of our coders to build ~~your organisation~~ a mobile web application for your organisation that delivers positive social impact.

Further on: 
> Web apps can be accessed by anyone using a web browser~~,~~ **-** on a computer, tablet or smartphone.

> We’ll ask you to join us weekly at the Founders and Coders’ space in North London...
- This section doesn't make it clear how the eight hours is divided, and the requirement for 8 hours comes out of the blue. Perhaps simply:
> Overall, you'll need to commit eight hours per week to work with us for the duration of the four-week programme, with your time spent both on and off site.
- And then the next question answers how that time is spent in more detail.

> You will participate in an introductory workshop that will help us find out more about your ~~needs~~**requirements**.

> If you want to continue the project ~~further~~, members of the development team...

> We are primarily looking for a committed product owner and an interesting problem to solve**,** with the potential to ~~make~~**have a** significant positive impact on ~~its~~ users.

Need to decide if we're going for 'Founders & Coders' or 'Founders and Coders' - current student website has the former. Also inconsistent capitalisation of the word 'programme'.

**Comment by sima-qian** [comment]
An additional question: are we keeping the 'What people have to say about working with us', 'Who we are', 'Portfolio', 'People we've worked with' and 'How we work' sections, and their associated copy?

**Comment by MissArray** [comment]
Lots of suggestions here - would you guys like to edit a copy of the copy (if that makes sense) for easy comparison? 

@sima-qian We were not asked to work on anything other than the text Dan's posted here, so not sure.

**Comment by MissArray** [comment]
1. Please take a couple of minutes to read through the re-edited draft of the text Dan's placed in his opening comment in this thread. Any suggestions, objections, please add a comment in this thread.
* If you do edit something in the online document, please flag the change you've made with a comment in that document to make it easier for everyone to spot it. 
* The entire document will be proofread (i.e., I'll proofread it) on Monday, once the copy has been finalised.
* The link (same doc):
https://docs.google.com/document/d/1yUGcobb06pzg76XcFrx3qbeiTH6GTg9j3EPH59PQYF4/edit?usp=sharing

2. For easy reference, I compiled a basic style manual. It's up to the team who'll deal with the design features of the student website to adjust and complete it:
https://docs.google.com/document/d/1gBT1xeacH7PzxpCYRPMhyyMg7xi8G62vAfVmn67EQTQ/edit?usp=sharing

**Comment by sima-qian** [comment]
Thanks for putting together a re-edited draft, @MissArray. I've had a look through and made some suggestions. We're hoping to have a final draft ready for Monday afternoon -- so if you could incorporate changes by then, that would be brilliant.

With regard to the style manual, we'll apply as much of it as we can to the current site, and hopefully incorporate more if & when we redesign the website.

**Comment by sofer** [comment]
@sima-qian we do not have to keep any of the existing sections that do not contribute to the message. I am happy for the editorial group to make a call on that.

**Comment by sima-qian** [comment]
@sofer Okay, thanks 👍
